src/components/utils/utils.js

```diff
@@ -157,3 +157,40 @@ export function calculateSusceptance(distance, linearCapacity) {
         Math.pow(10, 6)
     );
 }
+
+export const replaceAllDefaultValues = (arrayParams, oldValue, newValue) => {
+    return (
+        arrayParams &&
+        arrayParams.reduce((accumulator, current) => {
+            return [
+                ...accumulator,
+                {
+                    ...current,
+                    defaultValue:
+                        current.defaultValue === oldValue
+                            ? newValue
+                            : current.defaultValue,
+                },
+            ];
+        }, [])
+    );
+};
+
+export const flatObject = (object) => {
+    let result = {};
+
+    const recursiveFlatten = (obj) => {
```

Comment:
Why do you need a recursive method here ?

Reply:
no need to this function anymore